conans/client/rest/remote_credentials.py

```diff
@@ -0,0 +1,65 @@
+import json
+import os
+import platform
+
+from jinja2 import Template
+
+from conan.api.output import ConanOutput
+from conans.client.userio import UserInput
+from conans.errors import ConanException
+from conans.util.files import load
+
+
+class RemoteCredentials:
+    def __init__(self, cache):
+        self._cache = cache
+        self._urls = {}
+        creds_path = os.path.join(cache.cache_folder, "credentials.json")
```

Comment:
Let's add a `conf` to let users define this file wherever they want. This value could be the default one for sure.

Reply:
Better wait for users to request it?